tests/inspect/inspect.go

```diff
@@ -126,6 +127,16 @@ func main() {
 		os.Exit(1)
 	}
 
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, syscall.SIGTERM)
+	go func() {
+		for sig := range c {
+			if sig == syscall.SIGTERM {
```

Comment:
Unnecessary to check for SIGTERM since signal.Notify will only send that one.

```go
terminateCh := make(chan os.Signal, 1)
signal.Notify(terminateCh, syscall.SIGTERM)
go fun() {
  <- terminateCh
  os.Exit(1)
}
```
maybe?